content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,129 +1,103 @@
-is left for Git and its hosting platforms (e.g. GitHub, GitLab) to handle. These
-are designed for source code management (SCM) however, and thus ill-equipped to
-support data science needs. That's where DVC comes in: with its built-in data
-<abbr>cache</abbr>, reproducible [pipelines](/doc/start/data-pipelines), among
-several other novel features (see [Get Started](/doc/start/) for a primer.)
-
-## Track data and models for versioning
-
-Let's say you have an empty <abbr>DVC repository</abbr> and put a dataset of
-images in the `images/` directory. You can start tracking it with `dvc add`.
-This generate a `.dvc` file, which can be committed to Git in order to save the
-project's version:
-
-```dvc
-$ ls images/
-0001.jpg 0002.jpg 0003.jpg 0004.jpg ...
-
-$ dvc add images/
-
-$ git add images.dvc .gitignore
-$ git commit -m "Track images dataset with DVC."
-```
-
-DVC's also allows to define the processes that build artifacts based on tracked
-data, such as an ML model, by writing a simple `dvc.yaml` file that connects the
-pieces together:
-
-> `dvc.yaml` files can be written manually or generated with `dvc run`.
-
-```yaml
-stages:
-  train:
-    cmd: python train.py images/
-    deps:
-      - images
-    outs:
-      - model.pkl
-```
-
-> See [Data Pipelines](/doc/start/data-pipelines) for a comprehensive intro to
-> this feature.
-
-`dvc repro` can now execute the `train` stage for you. DVC will track all of its
-outputs (`outs`) automatically. Let's do that, and commit this project version:
+[Version control](https://en.wikipedia.org/wiki/Version_control) has become a
+staple in software engineering because it allows effective collaboration on
+source code. This means having a change history to traverse (commits),
+developing parallel features (branching and merging), peer-reviews (pull
+requests), release management, etc. Imagine using these features for data
+engineering and machine learning!
+
+Unfortunately, versioning tools like [Git](https://git-scm.com/) are designed to
+handle small text files. DVC begins by enabling the [tracking](#how-it-looks) of
+large datasets and other <abbr>data artifacts</abbr> by replacing them with
+small, human-readable [metafiles](/doc/user-guide/dvc-files-and-directories)
+that `git` can handle. The data itself is <abbr>cached</abbr> outside of Git.
+Other features of DVC build upon this foundation to tackle hurdles specific to
+data science.
+
+> 💡 Please see [Get Started](/doc/start) for a primer on all of DVC's features.
+
+Another limitation of source code management is storage. While some data and
+binary assets can exist in the Git repository, it's not usually recommended. And
+storage is severely limited by hosting services
+[like GitHub](https://docs.github.com/en/github/managing-large-files/what-is-my-disk-quota).
+DVC addresses this problem by integrating with
+[dedicated storage](#versioned-storage) platforms.
```

Comment:
we need some image at the very top

Reply:
Yep, still working on that. Will send over a sketch ASAP.

Reply:
So here's an idea that tries to show a disorganized folder above ("before") and what it looks as different workspace versions below ("after")

![IMG_20201012_205412](https://user-images.githubusercontent.com/1477535/95808136-678e2880-0cd1-11eb-97cd-d60bdbd79cc0.jpg)

Cons:
- Too much info for a single figure, probably (could be summarized to less files above and only 3 versions below.
- Focuses mainly on one of the benefits mainly (better organization, efficient storage) although it also implies the enabling of git workflow.

Reply:
yep, it feels like too much text

Reply:
How about just the bottom half?

![image](https://user-images.githubusercontent.com/1477535/96044463-121f5c00-0e36-11eb-8128-a5c2d6fb4aa1.png)

I guess without the (before) part it may not make much sense... I'll think of something else ⏳